docs/superpowers/specs/2026-05-17-recording-reminder-matrix-mockup.html

```
@@ -0,0 +1,161 @@
+<h2>Final matrix · recording / reminder / processing / default reminder</h2>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)